docs/user/debugging.md

```diff
@@ -35,27 +42,28 @@ adb shell am start -n org.mozilla.samples.glean/mozilla.telemetry.glean.debug.Gl
 > **Note:** In previous versions of Glean the activity was available with the name `mozilla.components.service.glean.debug.GleanDebugActivity`.
 >
 > If you're debugging an old build, try running:
-> ```
+>
+> ```shell
 > adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean.debug.GleanDebugActivity \
 >   --ez logPings true \
 >   --es sendPing metrics \
 >   --es tagPings test-metrics-ping
 > ```
 
-### Important GleanDebugActivity notes!
+### Important GleanDebugActivity notes
```

Comment:
This part applies to both the ios and the android part, as it depicts the two different workflows and it's not specifically tied to Android.